src/Record/Record.Model/Records.csproj

```
@@ -12,7 +12,7 @@
 		<EnvironmentSuffix></EnvironmentSuffix>
 		<ReleaseType></ReleaseType>
 		<PackageId>Record</PackageId>
-		<VersionPrefix>1.1.1$(ReleaseType)</VersionPrefix>
+		<VersionPrefix>1.1.2$(ReleaseType)</VersionPrefix>
```

```suggestion
		<VersionPrefix>2.0.0$(ReleaseType)</VersionPrefix>
```

Since you have removed a method this is not backwards compatiable.
If you want to keep it to a minor version you could instead keep WithFileContent and point WithContent to it, and then mark WithContent as deprecated. So you communicate that this method is on its way out, but both are supported.